RecoLocalTracker/SiPixelClusterizer/python/SiPixelClusterizerPreSplitting_cfi.py

```diff
@@ -8,9 +8,6 @@
 siPixelClustersPreSplitting = _siPixelClusters.clone() 
 
 from Configuration.ProcessModifiers.siPixelDigiMorphing_cff import siPixelDigiMorphing
-siPixelDigiMorphing.toModify(
```

Comment:
I think this got broken by https://github.com/cms-sw/cmssw/pull/46853 in 15.0.X

Reply:
It surely did, sorry.